apps/src/templates/SchoolInfoInputs.jsx

```diff
@@ -125,6 +125,8 @@ export default class SchoolInfoInputs extends Component {
 
     return (
       <div style={{width: 600, ...this.props.styles}}>
+        <h5>{i18n.schoolInformationHeader()}</h5>
```

Comment:
nit: use `Heading5` here

Reply:
There was no Heading5 in that file, but that's no longer an issue since the styling I should be using is t-shirt sizing anyway😅

Reply:
Because this is on the old/existing flow, I'd rather not change the styling. These lines are just moved from the file they were rendered in before. Does that make sense? Heading5 looks fairly different from the built in h5

Reply:
(For more context- I moved these two lines into this component so that the new one could also have a formatted (and different string) title and it wouldn't be duplicated across the AB test

Reply:
yep thats fine!